### PR TITLE
Fix spelling, wording and updated number formatting

### PR DESCRIPTION
**Pull request by MyIgel**
I updated most pages to
* Fix some wording/spelling issues
* Use small space before units
* Update number formatting to be more consistent
* And generally make the page friendlier / the docs better readable and understandable

### CODE REVIEW

File: content/docs/classic/pcb/_index.de.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ description: >
 Ein sehr wichtiger Bestandteil des Gerätes ist die Platine, auch
 &bdquo;PCB&rdquo; (englisch _printed circuit board_) genannt. Der schwierige
 Teil hieran ist das &bdquo;printed&rdquo; -- ein PCB ist ein professionell
-hergestelltes Bauteil das schwierig selbst zu erstellen ist. Hier sind deine
+hergestelltes Bauteil, dass schwierig selbst zu erstellen ist. Hier sind deine
 Optionen:
```
Comment (gluap) [inline content/docs/classic/pcb/_index.de.md:17]: das Bauteil

File: content/device.de.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ menu:
 
 # Das Messgerät
 
-Kern unseres Projekts ist ein Gerät, das die Überholabstände beim Radfahren
+Kern unseres Projektes ist ein Gerät, dass die Überholabstände beim Radfahren
 aufzeichnet. Dieses Gerät haben wir als quelloffenes Design entwickelt, sodass
```
Comment (gluap) [inline content/device.de.md:15]: das Gerät

**Review by gluap — APPROVED** [review]
I really liked almost all of the changes. Unfortunately my test of github codespaces went not so well, so my suggested changes are now in separate branch on my own fork - I'll approve and merge here, and then immediately merge the very few changes to this PR).

Thank you for being so thorough and really going through all this text!

File: content/docs/classic/build-instructions/_index.de.md [hunk 3]
```diff
@@ -960,8 +966,8 @@ beide Varianten ermöglichen, kannst du dir die Öffnung frei aussuchen.
     herausfallen.
   - Das Kabel sollte außen um die Elektronik herumgeführt werden, *nicht*
     zwischen dem GPS- und SD-Karten-Modul hindurch. Im neuen Gehäuse sollte
-    dort genug Platz sein.
-  - Jetzt zeigt sich auch, warum ca. 5cm Abisolierung hier sinnvoll ist, denn
+    dafür genug Platz sein.
+  - Jetzt zeigt sich auch, warum ca. 5&thinsp;cm Abisolierung hier sinnvoll ist, denn
     ohne Isolierung ist das Kabel flexibler.
```
[the remark this answers is not in the document]
Comment (MyIgel) [reply content/docs/classic/build-instructions/_index.de.md:971]: Andererseits sind die 7 (glaube ich) auch wiederum etwas viel Kabel das potentiell störungen verursachen kann 😄

**Comment by MyIgel** [comment]
Happy to help (and I am pretty sure that I have not found everything that could be fixed)

**Pull request merged**